Added Accrualify section (#574)
mattermost/mattermost-handbook: operations/finance/spending-company-money/how-to-use-accrualify/how-to-set-up-your-account.md

## How to set up your account

The Finance team will send you a Welcome email from Accrualify if you need to do any of the following:

> * Submit a Purchase Order

Reviewer: ```suggestion
* Submit a purchase order
```

> * Review a Purchase Order

Reviewer: ```suggestion
* Review a purchase order
```

* Review a vendor invoice

> If you need to submit a purchase order, please make sure you've submitted a budget request. Once your [budget request](https://handbook.mattermost.com/operations/finance/spending-company-money/budget) has been approved, the Finance team will provision you on Accrualify. You should receive a “Welcome to Accrualify” email, which includes a link to set up your account.

Reviewer: ```suggestion
If you need to submit a purchase order, please make sure you've submitted a [budget request](https://handbook.mattermost.com/operations/finance/spending-company-money/budget). Once your budget request has been approved, the Finance team will provision you an account on Accrualify. You should receive a “Welcome to Accrualify” email, which includes a link to set up your account.
```

![](../../../../.gitbook/assets/How-to-set-up-your-Accrualify-account-1.PNG)

> Once you have set up your account, you will be able to view your Dashboard. From your Dashboard, you will be able to view all your Actionable Items.

Reviewer: ```suggestion
Once you have set up your account, you can log in to see your Accrualify dashboard to review and take action on your actionable items.
```
